apps/core/components/priceSummary/PriceSummary.js

```diff
@@ -22,15 +17,15 @@ type Props = {|
 |};
 type State = {|
   +expanded: boolean,
-  +opacity: AnimatedValue,
+  +opacity: any, // @TODO - set more exact type
```

Comment:
@jvaclavik Any idea about this one? Animation related types are not yet exported from react-native (in version we are using) and if I just quickly tried import it directly like below it generates few new errors.
```
import type AnimatedValue from 'react-native/Libraries/Animated/src/nodes/AnimatedValue';
```

Reply:
Hmm, it's very hard to find something relevant about flow types for `AnimatedValue`. Your solution looks fine, what kind of errors does it produce?

Reply:
Ok, so I'll keep todo comments for now. And we will update it later, after expo let us upgrade to some higher version which have it exported. 
Mentioned import causes multiple errors related to missing props on AnimatedValue.

Reply:
I will commit my solution to this PR.